systest/bulk_live/live/docker-compose.yml

```diff
@@ -16,21 +16,18 @@ services:
       target: /gobin
       read_only: true
     command: /gobin/dgraph alpha --my=alpha1:7080 --zero=zero1:5080 --logtostderr
-      -v=2 --raft='idx=1; group=1;' 
+      -v=2 --raft='idx=1; group=1;'
       --security "whitelist=10.0.0.0/8,172.16.0.0/12,192.168.0.0/16;"
     deploy:
       resources:
         limits:
           memory: 32G
-  minio1:
+  minio:
     image: minio/minio:RELEASE.2020-11-13T20-10-18Z
     ports:
-    - "9001"
-    volumes:
-    - type: bind
-      source: .
-      target: /data/minio
-      read_only: false
+      - "9001"
+    env_file:
```

Comment:
This is confusing to me. Why do we need to pass `env file` here when we add env: https://github.com/dgraph-io/dgraph/pull/8569/files#diff-35cf19c47058f248f8d5b1ac504286a97aa78bc5bfa461f97aea3ceca89d664bR137

And we add the same values which are present in the env file. 

I'd rather not set the `env` file and just do https://github.com/dgraph-io/dgraph/pull/8569/files#diff-35cf19c47058f248f8d5b1ac504286a97aa78bc5bfa461f97aea3ceca89d664bR137. If that's a possibility. I haven't checked it yet though.

Reply:
There are 2 reference points here https://github.com/dgraph-io/dgraph/blob/aman/minio/systest/backup.env && https://github.com/dgraph-io/dgraph/pull/8569/files#diff-35cf19c47058f248f8d5b1ac504286a97aa78bc5bfa461f97aea3ceca89d664bR137.

I think I understand why @mangalaman93 is doing it this way in the docker compose file [here]( https://github.com/dgraph-io/dgraph/blob/aman/minio/systest/backup.env) - if my understanding is right then he is loading the `env_file` to force minio service to use those keys for access validations from client side. Let me know if this assumption is correct.

The second [url](https://github.com/dgraph-io/dgraph/pull/8569/files#diff-35cf19c47058f248f8d5b1ac504286a97aa78bc5bfa461f97aea3ceca89d664bR137) is showing how he accesses the minio service - the hardcoded values there looks a little confusing to me as well. Would it be possible for you to read the `backup.env` file and load the same into the env? (it will be a single source of truth this way)

Reply:
We run bulk loader and live loader as part of the test. We need to setup env for the processes running these loaders. Beyond that, we setup env for the minio container as well.

Reply:
makes sense ^